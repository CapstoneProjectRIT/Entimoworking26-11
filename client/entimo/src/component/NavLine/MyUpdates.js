import React from 'react';
import Navbar from '../Dashboard/index';

const MyUpdates = () => (
  <>
    <Navbar />
    <div
      style={{
        display: 'flex',
        justifyContent: 'Right',
        alignItems: 'Right',
        height: '100vh',
      }}
    >
      <h1>These are my Upadates.</h1>
    </div>
  </>
);

export default MyUpdates;
