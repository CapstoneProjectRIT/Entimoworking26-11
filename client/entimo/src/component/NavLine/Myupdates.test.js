import React from 'react';
import { create } from 'react-test-renderer';
import MyUpdates from './MyUpdates';

describe('My first snapshot test', () => {
  test('testing app button', () => {
    const tree = create(<MyUpdates />);
    expect(tree.toJSON()).toMatchSnapshot();
  });
});
