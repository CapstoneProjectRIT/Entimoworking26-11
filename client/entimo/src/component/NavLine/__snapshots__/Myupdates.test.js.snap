// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`My first snapshot test testing app button 1`] = `
Array [
  <nav
    className="sc-bdnxRM tipZl"
  >
    <svg
      className="sc-dlnjwi NOtWA"
      fill="currentColor"
      height="1em"
      stroke="currentColor"
      strokeWidth="0"
      style={
        Object {
          "color": undefined,
        }
      }
      viewBox="0 0 448 512"
      width="1em"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        d="M16 132h416c8.837 0 16-7.163 16-16V76c0-8.837-7.163-16-16-16H16C7.163 60 0 67.163 0 76v40c0 8.837 7.163 16 16 16zm0 160h416c8.837 0 16-7.163 16-16v-40c0-8.837-7.163-16-16-16H16c-8.837 0-16 7.163-16 16v40c0 8.837 7.163 16 16 16zm0 160h416c8.837 0 16-7.163 16-16v-40c0-8.837-7.163-16-16-16H16c-8.837 0-16 7.163-16 16v40c0 8.837 7.163 16 16 16z"
      />
    </svg>
    <div
      className="sc-hKFxyN dVDJAw"
    >
      <div
        className="sc-gtsrHT dMSzPu"
      >
        Home
      </div>
      <div
        className="sc-gtsrHT dMSzPu"
      >
        Trending Task
      </div>
      <div
        className="sc-gtsrHT dMSzPu"
      >
        My Tasks
      </div>
      <div
        className="sc-gtsrHT dMSzPu"
      >
        My Updates
      </div>
    </div>
  </nav>,
  <div
    style={
      Object {
        "alignItems": "Right",
        "display": "flex",
        "height": "100vh",
        "justifyContent": "Right",
      }
    }
  >
    <h1>
      These are my Upadates.
    </h1>
  </div>,
]
`;
